pretty.go

```diff
@@ -27,11 +27,14 @@ func listPrettyPrint(w io.Writer, output []outputList) {
 
 func updatePrettyPrint(w io.Writer, output []outputUpdates) {
 	table := tablewriter.NewWriter(w)
-	table.SetHeader([]string{"Update?", "Name", "Constraint", "Version", "Latest matching", "Latest"})
+	table.SetHeader([]string{"Update?", "Name", "Constraint", "Version", "Latest matching", "Latest overall"})
```

Comment:
i think this is ok as-is, since we already have "latest matching". This output type should be as concise as possible by default (think of e.g. terminal width limits)

Reply:
@sgreben I did it because it seemed a better naming style for everybody but I can reverse the change if you like. I saw some "not so good naming" in code too, but I left it for the next PR. So what do you say?